contracts/BC_fusion/System.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.17;
+
+contract System {
+    address public constant VALIDATOR_CONTRACT_ADDR = 0x0000000000000000000000000000000000001000;
+    address public constant SLASH_CONTRACT_ADDR = 0x0000000000000000000000000000000000001001;
```

Comment:
can we change the "public" here to "private" or "internal", so that there wont have any function that wrap for this public variable, it can reduce the code size.

Reply:
Done